src/Features/Core/Portable/Completion/CommonCompletionProvider.cs

```diff
@@ -29,7 +31,7 @@ public override bool ShouldTriggerCompletion(SourceText text, int position, Comp
             }
         }
 
-        internal virtual bool IsInsertionTrigger(SourceText text, int insertedCharacterPosition, OptionSet options)
+        public virtual bool IsInsertionTrigger(SourceText text, int insertedCharacterPosition, OptionSet options)
```

Comment:
❔ Why make this public?

Reply:
because it's in an internal type.  that's the general pattern we use.